test/widgets/autocomplete_test.dart

```diff
@@ -77,34 +89,59 @@ void main() {
       await tester.enterText(composeInputFinder, 'hello @user ');
       await tester.enterText(composeInputFinder, 'hello @user t');
       await tester.pumpAndSettle(); // async computation; options appear
+
       // "User Two" and "User Three" appear, but not "User One"
       check(tester.widgetList(find.text('User One'))).isEmpty();
+      final user1AvatarFinder =
```

Comment:
Added the tests for this feature in the previous test block with the motivation that the user name and avatar are closely related to each other.